Update docs on OpenAPI-related metadata (#32624)
dotnet/AspNetCore.Docs: aspnetcore/fundamentals/minimal-apis/includes/openapi9.md

[!code-xml[](~/fundamentals/minimal-apis/9.0-samples/WebMinOpenApi/projectFile.xml?highlight=10)]

> To learn more about the `Microsoft.AspNetCore.OpenApi` package, please review the docs <xref:fundamentals/minimal-apis/aspnetcore-openapi>.

Reviewer: ```suggestion
For more information, see <xref:fundamentals/minimal-apis/aspnetcore-openapi>.
```

## Describe endpoints

> OpenAPI supports providing summaries and descriptions of routes that are registered in an application. Minimal APIs supports two strategies for setting these properties on a given endpoint:

Reviewer: ```suggestion
OpenAPI supports providing summaries and descriptions of routes that are registered in an application. Minimal APIs support four strategies for setting these properties on a given endpoint:
```

* Via the <xref:Microsoft.AspNetCore.Http.OpenApiRouteHandlerBuilderExtensions.WithSummary%2A> extension method on the endpoint for summaries
* Via the <xref:Microsoft.AspNetCore.Http.OpenApiRouteHandlerBuilderExtensions.WithDescription%2A> extension method on the endpoint for descriptions
* Via the <xref:Microsoft.AspNetCore.Http.EndpointSummaryAttribute> on the endpoint
* Via the <xref:Microsoft.AspNetCore.Http.EndpointDescriptionAttribute> on the endpoint

> The following sample demonstrates the different strategies for setting summaries and descriptions.

Reviewer: ```suggestion
The following sample demonstrates two strategies for setting summaries and descriptions.
```

```csharp
app.MapGet("/extension-methods", () => "Hello world!")
  .WithSummary("This is a summary.")
  .WithDescription("This is a description.");

app.MapGet("/attributes",
  [EndpointSummary("This is a summary.")]
  [EndpointDescription("This is a description.")]
  () => "Hello world!");
```

## Set OpenAPI tags

> OpenAPI supports specifying tags on each endpoint as a form of categorization. Minimal APIs supports two strategies for setting OpenAPI tags on a given endpoint:

Reviewer: ```suggestion
OpenAPI supports specifying tags on each endpoint as a form of categorization. Minimal APIs support two strategies for setting OpenAPI tags on a given endpoint:
```

* Via the <xref:Microsoft.AspNetCore.Http.OpenApiRouteHandlerBuilderExtensions.WithTags%2A> extension method
* Via the <xref:Microsoft.AspNetCore.Http.TagsAttribute>

> The follow sample demonstrates the different strategies for setting tags.

Reviewer: ```suggestion
The following sample demonstrates the different strategies for setting tags.
```

```csharp
app.MapGet("/extension-methods", () => "Hello world!")
  .WithTags("todos", "projects");

app.MapGet("/attributes",
  [Tags("todos", "projects")]
  () => "Hello world!");
```

## Excluding endpoints from the generated document

> By default, all endpoints that are defined in an application are documented in the generated OpenAPI file. Minimal APIs supports two strategies for excluding a given endpoint from the OpenAPI document:

Reviewer: ```suggestion
By default, all endpoints that are defined in an application are documented in the generated OpenAPI file. Minimal APIs support two strategies for excluding a given endpoint from the OpenAPI document:
```
